www/class/centreon-clapi/centreonInstance.class.php

```diff
@@ -126,6 +136,14 @@ public function initUpdateParameters($parameters)
             throw new CentreonClapiException(self::MISSINGPARAMETER);
         }
 
+        // Check IPv6, IPv4 and FQDN format
+        if ($params[1] == 'ns_ip_address' && !preg_match('/^([0-9a-fA-F]{4}|0)(\:([0-9a-fA-F]{4}|0)){7}$/', $params[2])
```

Comment:
there are already FILTER_VALIDATE_DOMAIN and FILTER_VALIDATE_IP which can be used with `filter_var`

Reply:
done